src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2426,6 +2426,20 @@ def aks_nodepool_snapshot_create(cmd,    # pylint: disable=too-many-locals,too-m
     return client.create_or_update(resource_group_name, snapshot_name, snapshot, headers=aks_custom_headers)
 
 
+def aks_nodepool_snapshot_update(cmd, client, resource_group_name, name, tags):   # pylint: disable=unused-argument
```

Comment:
```suggestion
def aks_nodepool_snapshot_update(cmd, client, resource_group_name, snapshot_name, tags):   # pylint: disable=unused-argument
```
should match with the declaration in `_params.py` otherwise the hep message will prompt incorrect information. Please update the usage of this variable in the function accordingly.

![image](https://github.com/Azure/azure-cli-extensions/assets/81607949/143e70c6-c235-4984-90b7-3fe5c2e6875f)